tests/drivers/misc/stm32_tsc/boards/stm32u083c_dk.overlay

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2024 Arif Balik <arifbalik@outlook.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	zephyr,user {
+		/* This will be used to trigger the SYNC pin high and low programmatically.
+		 * They have to be connected.
+		 */
+		signal-gpios = <&gpioa 10 GPIO_ACTIVE_HIGH>;
+	};
+};
+
+&tsc {
+	status = "okay";
+
+	pinctrl-0 = <&tsc_g1_io1_pb12 &tsc_g1_io2_pb13 &tsc_g6_io1_pd10
+					&tsc_g6_io2_pd11 &syncpin>;
+	pinctrl-names = "default";
+
+	pulse-generator-prescaler = "TSC_PG_PRESC_DIV128";
+
+	spread-spectrum;
+	spread-spectrum-prescaler = <2>;
+	spread-spectrum-deviation = <10>;
+
+	max-count-value = "TSC_MCV_16383";
+
+	synced-acquisition;
+	syncpol-rising;
+};
+
+&pinctrl {
+	syncpin: syncpin_pd2 {
+		pinmux = <STM32_PINMUX('D', 2, AF9)>;
+		drive-open-drain;
+	};
```

Comment:
Why isn't this one provided through -pintrl.dtsi files ?

Reply:
ah, my bad